packages/statemanager/src/types.ts

```diff
@@ -0,0 +1,4 @@
+/**
+ * Log that the contract emits.
+ */
+export type Log = [address: Buffer, topics: Buffer[], data: Buffer]
```

Comment:
This `Log` type seems a bit exotic to me here, does this really belong in here? Especially since `StateManager` isn't even an event emitter?

If I am not overlooking somthing I would have a strong tendency to move back to VM, otherwise let me know what context I might be missing.